src/System.Runtime.Extensions/ref/System.Runtime.Extensions.cs

```diff
@@ -1661,6 +1656,101 @@ public static partial class WebUtility
         public static byte[] UrlEncodeToBytes(byte[] value, int offset, int count) { throw null; }
     }
 }
+namespace System.Reflection
+{
+    public partial class AssemblyNameProxy : System.MarshalByRefObject
+    {
+        public AssemblyNameProxy() { }
+        public System.Reflection.AssemblyName GetAssemblyName(string assemblyFile) { throw null; }
+    }
+}
+namespace System.Runtime
+{
+    public static partial class ProfileOptimization
+    {
+        public static void SetProfileRoot(string directoryPath) { }
+        public static void StartProfile(string profile) { }
+    }
+}
+namespace System.Runtime.CompilerServices
+{
+    public sealed partial class SwitchExpressionException : System.InvalidOperationException
+    {
+        public SwitchExpressionException() { }
+        public SwitchExpressionException(System.Exception innerException) { }
+        public SwitchExpressionException(object unmatchedValue) { }
+        public SwitchExpressionException(string message) { }
+        public SwitchExpressionException(string message, System.Exception innerException) { }
+        public override string Message { get { throw null; } }
```

Comment:
This was missing from the ref.
https://github.com/dotnet/corefx/pull/34954

cc @maryamariyan